yandex_music/track_short.py

```diff
@@ -21,10 +26,10 @@ class TrackShort(YandexMusicObject):
     """
 
     def __init__(self,
-                 id,
-                 timestamp,
-                 album_id=None,
-                 client=None,
+                 id: str,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'id'](https://app.codacy.com/app/MarshalX/yandex-music-api/pullRequest?prid=4731841)